lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -1671,7 +1719,14 @@ tokenize_sigil_contents([H | T] = Original, [S | _] = SigilName, Line, Column, S
   case elixir_interpolation:extract(Line, Column + 1, Scope, ?is_downcase(S), T, sigil_terminator(H)) of
     {NewLine, NewColumn, Parts, Rest, NewScope} ->
       Indentation = nil,
-      add_sigil_token(SigilName, Line, Column, NewLine, NewColumn, tokens_to_binary(Parts), Rest, NewScope, Tokens, Indentation, <<H>>);
+      try
+        add_sigil_token(SigilName, Line, Column, NewLine, NewColumn, tokens_to_binary(Parts), Rest, NewScope, Tokens, Indentation, <<H>>)
+      catch
+        error:#{'__struct__' := 'Elixir.UnicodeConversionError', message := Message} ->
+          Sigil = [$~, S, H],
+          Message = " (for sigil ~ts starting at line ~B)",
+          interpolation_error(Message, [$~] ++ SigilName ++ Original, Scope, Tokens, Message, [Sigil, Line], Line, Column, [H], [sigil_terminator(H)])
```

Comment:
Shouldn't there be a test added for this change? Dialyzer [says](https://github.com/michallepicki/elixir-lang-dialyzer-runs/actions/runs/16557132698/job/46820247947) this function call will fail, the first argument of `interpolation_error` should be a [5 element tuple](https://github.com/lukaszsamson/elixir/blob/29973cb07309c08ebd4ae7a4f727a10f0a5d4c48/lib/elixir/src/elixir_tokenizer.erl#L1465)

Reply:
Try catch is not needed here. It calls `tokens_to_binary` which may throw on liststring but it will only get binary args here